ONSdigital/dp-dataset-api
# Feature/generate downloads on publish

### What

- Re-generate downloads when version is published

### How to review

- Publish a version
- Verify a kafka message is sent to generate downloads

note acceptance tests will be written once the exporters can handle the download message correctly

### Who can review

Anyone
